src/dotty/tools/dotc/core/TypeErasure.scala

```diff
@@ -51,10 +55,19 @@ object TypeErasure {
       false
   }
 
-  case class ErasedValueType(cls: ClassSymbol, underlying: Type) extends CachedGroundType {
+  abstract case class ErasedValueType(cls: ClassSymbol, underlying: Type)
+  extends CachedGroundType with ValueType {
```

Comment:
I'd say it needs documentation. And I'd like it to include why it's a `GroundType`, but not a `ProxyType`.